packages/atomic/src/components/search/atomic-refine-modal/atomic-refine-modal.tsx

```diff
@@ -209,6 +249,17 @@ export class AtomicRefineModal implements InitializableComponent {
           )}
         </div>
         <slot name="facets"></slot>
+        <div
+          class="flex flex-col"
+          style={{
+            gap: 'var(--atomic-refine-modal-facet-margin, 20px)',
+            marginTop: isFacetElements
+              ? 'var(--atomic-refine-modal-facet-margin, 20px)'
+              : '',
+          }}
+        >
+          {automaticFacets}
+        </div>
```

Comment:
This is the same style that is used for the normal facets.

[link](https://github.com/coveo/ui-kit/blob/4a0965688b83e211ed47efcd7e7c8912747f2663/packages/atomic/src/components/common/refine-modal/refine-modal-common.tsx#L113)